### PR TITLE
Add shortcuts for windows & linux to debugger.html

### PR DESCRIPTION
**Pull request by mdamien**
Very simple PR to add the shortcuts for windows/linux to the web debuger interface so it's less mac-centric.

**Test plan**: Open the debugger and see the shorcuts for windows/linux/mac

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @frantic and @chrisnager to be potential reviewers.

**Comment by janicduplessis** [comment]
I'd be cool to do a simple navigator.userAgent check and show only the shortcut for the current platform. We can do a simple regex check.

File: local-cli/server/util/debugger.html [hunk 1]
```diff
@@ -127,7 +135,7 @@
     <p>
       React Native JS code runs inside this Chrome tab.
     </p>
-    <p>Press <kbd class="shortcut">⌘⌥J</kbd> to open Developer Tools. Enable <a href="http://stackoverflow.com/a/17324511/232122" target="_blank">Pause On Caught Exceptions</a> for a better debugging experience.</p>
+    <p>Press <kbd id='dev_tools_shortcut' class="shortcut">⌘⌥J</kbd></kbd> to open Developer Tools. Enable <a href="https://stackoverflow.com/a/17324511/232122" target="_blank">Pause On Caught Exceptions</a> for a better debugging experience.</p>
```
Comment (satya164) [inline local-cli/server/util/debugger.html]: There seems to be an extra `</kbd>` here

Comment (mdamien) [inline local-cli/server/util/debugger.html]: oops, nice catch

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-0** [comment]
@mdamien updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9812/files/f5ca936c39466144630a33de41b4fc33d90942fa..b51d26348441b7c14dc46aa2cf45a7f0fe8d0364)

**Comment by facebook-github-bot-8** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3862570) to review internal test results.

**Pull request closed without merging**